Jenkinsfile

```diff
@@ -0,0 +1,23 @@
+properties([[
```

Comment:
consider refactoring to declarative pipeline for consistency sake

Reply:
Yes. Declarative only for edgex